CONTRIBUTING.md

```diff
@@ -185,37 +185,4 @@ By making a contribution to this project, I certify that:
 
 ## Code of Conduct
 
-This Code of Conduct is adapted from [Rust's wonderful
-CoC](http://www.rust-lang.org/conduct.html).
-
-* We are committed to providing a friendly, safe and welcoming
-  environment for all, regardless of gender, sexual orientation,
-  disability, ethnicity, religion, or similar personal characteristic.
-* Please avoid using overtly sexual nicknames or other nicknames that
-  might detract from a friendly, safe and welcoming environment for
-  all.
-* Please be kind and courteous. There's no need to be mean or rude.
-* Respect that people have differences of opinion and that every
-  design or implementation choice carries a trade-off and numerous
-  costs. There is seldom a right answer.
-* Please keep unstructured critique to a minimum. If you have solid
-  ideas you want to experiment with, make a fork and see how it works.
-* We will exclude you from interaction if you insult, demean or harass
-  anyone. That is not welcome behavior. We interpret the term
-  "harassment" as including the definition in the [Citizen Code of
-  Conduct](http://citizencodeofconduct.org/); if you have any lack of
-  clarity about what might be included in that concept, please read
-  their definition. In particular, we don't tolerate behavior that
-  excludes people in socially marginalized groups.
-* Private harassment is also unacceptable. No matter who you are, if
-  you feel you have been or are being harassed or made uncomfortable
-  by a community member, please contact one of the channel ops or any
-  of the TC members immediately with a capture (log, photo, email) of
-  the harassment if possible. Whether you're a regular contributor or
-  a newcomer, we care about making this community a safe place for you
-  and we've got your back.
-* Likewise any spamming, trolling, flaming, baiting or other
-  attention-stealing behavior is not welcome.
-* Avoid the use of personal pronouns in code comments or
-  documentation. There is no need to address persons when explaining
-  code (e.g. "When the developer").
+[Please read it.](./CODE_OF_CONDUCT.md)
```

Comment:
I think the reason the CoC was originally included in `CONTRIBUTING.md` is that this file is automatically linked by GitHub on the "open a PR" form. I might be imagining this, but I think there was some legal reason that that started happening, too, related to the developer's certificate of origin. Given that, it might be good to note to the readers coming in from that link that the CoC is a binding part of contributing to Node:

 ```markdown
By submitting a pull request to, issue to, or commenting on a pull request or issue on any 
of the "nodejs/" repositories, or using the #node.js or #io.js IRC channels, you agree to 
abide by the Node.js [Code of Conduct](./CODE_OF_CONDUCT.md).
```

I'd rather not hold up the PR for that, though — it can be done separately.